workloads/logging/common.sh

```diff
@@ -33,56 +33,56 @@ deploy_logging_stack() {
   ./deploy_logging_stack.sh
 }
 
+
 run_workload() {
-  timestamp=`date "+%d-%m-%YT%H:%M:%S"`
+  timestamp="$(date "+%d-%m-%YT%H:%M:%S")"
+  export CLUSTER_ID="$(oc get infrastructure.config.openshift.io cluster -o json 2>/dev/null | jq -r .status.infrastructureName)"
   log "Customizing log-generator CR file"
-  envsubst < files/log_generator.yaml > log_generator_$timestamp.yaml
-  if [[ ${DEPLOY_LOGGING} == "true" ]]; then
-    # Get bearer token and ES url if applicable
-    if [[ ${CUSTOM_ES_URL} == "" ]]; then
-      ES_BACKEND_TOKEN=`oc whoami -t`
-      ES_BACKEND_URL=`oc get route elasticsearch -n openshift-logging -o jsonpath={.spec.host}`
-    else
-      ES_BACKEND_URL=$CUSTOM_ES_URL
-    fi
+  envsubst < files/log_generator.yaml > temp/log_generator_"${timestamp}".yaml
```

Comment:
does this create the temp dir if it doesnt exist?

Reply:
good catch, adding creation of the folder